rtengine/procparams.cc

```diff
@@ -129,22 +131,22 @@ RetinexParams::RetinexParams ()
     setDefaults ();
 }
 
-void RetinexParams::getDefaulttransmissionCurve(std::vector<double> &curve)
+void RetinexParams::getDefaulttransmissionCurve (std::vector<double> &curve)
```

Comment:
@Beep6581 
When your commit is finished, I will make the (important) changes desired by Elle.

At least
* CAT02 for Viewing conditions (slider with autobutton)
* Temp scene
* green scene
* perhaps change "Yb" scene (actually in option) to main menu - perhaps with a slider and autobutton

Modifications to ciecam02.cc
Modifications to "ciecam_02float" and "ciecam_02" in improcfun.cc
and obviously all linked files (paramsedited, procparams, procevents, refreshmap,...)